apps/builder/app/builder/features/style-panel/controls/color/color-control.tsx

```diff
@@ -3,12 +3,26 @@ import { toValue } from "@webstudio-is/css-engine";
 import type { ControlProps } from "../../style-sections";
 import { ColorPicker } from "../../shared/color-picker";
 import { colord } from "colord";
+import { getStyleSource } from "../../shared/style-info";
+import { styleConfigByName } from "../../shared/configs";
+import type { StyleValue } from "@webstudio-is/css-data";
+import type { IntermediateStyleValue } from "../../shared/css-value-input";
+import { useState } from "react";
 
 export const ColorControl = ({
```

Comment:
Think about how a color picker with the same logic could be reused with props panel

Reply:
Probably can't as of now it will be simple wrapper around CSSValueInput because CSSValue input already has most logic we need.

Reply:
Also for what attribute we will need color picker?

Reply:
CssValueInput should also become usable over props, why would we miss an opportunity in a component to allow passing some CSS property?

Reply:
we already support color control in props, storybook supports it too, any property can choose to accept a color, there is absolutely 0 doubt about the need for it